content/_code-samples/non-fungible-token/py/create-sell-offer-nft.py

```diff
@@ -0,0 +1,86 @@
+from xrpl.models.transactions.nftoken_create_offer import NFTokenCreateOffer, NFTokenCreateOfferFlag
+from xrpl.transaction import safe_sign_and_autofill_transaction, send_reliable_submission
+from xrpl.models.requests import AccountNFTs
+from xrpl.clients import JsonRpcClient
+from xrpl.models import NFTSellOffers
+from xrpl.wallet import Wallet
+
+# Put up a NFT sell offer on the XRPL via NFTokenCreateOffer transaction with flag TF_SELL_NFTOKEN
+# https://xrpl.org/nftokencreateoffer.html#nftokencreateoffer
+
+# For this code snippet to work, you need import an Account Seed with an already issued NFT
+# You could use 'mint_nft.py' on the same folder to import an account
+
+seed = ""
+
+# Connect to a testnet node
+print("Connecting to Testnet...")
+JSON_RPC_URL = "https://s.altnet.rippletest.net:51234/"
+client = JsonRpcClient(JSON_RPC_URL)
+
+# Initialize wallet from seed
+issuer_wallet = Wallet(seed=seed, sequence=0)
+issuerAddr = issuer_wallet.classic_address
+
+print(f"\nIssuer Account: {issuerAddr}")
+print(f"          Seed: {issuer_wallet.seed}")
+
+# Query the issuer account for its NFTs
+get_account_nfts = client.request(AccountNFTs(account=issuerAddr))
+
+nft_int = 1
+print(f"\n - NFTs owned by {issuerAddr}:")
+for nft in get_account_nfts.result['account_nfts']:
+    print(f"\n{nft_int}. NFToken metadata:"
+          f"\n    NFT ID: {nft['NFTokenID']}"
+          f"\n    Issuer: {nft['Issuer']}"
+          f"\n NFT Taxon: {nft['NFTokenTaxon']}")
+    nft_int += 1
+
+
+nftoken_id = input("\nEnter which NFT (by ID) you want to put up for sell: ")
+nftoken_amount = input("Enter how much you want the NFT to go for, in drops: ")
+
+# Sell the NFT on the open market
+sell_tx = NFTokenCreateOffer(
+    account=issuerAddr,
+    nftoken_id=nftoken_id,
+    amount=nftoken_amount,  # 10 XRP in drops, 1 XRP = 1,000,000 drops
```

Comment:
Could be any amount since it's user input.
```suggestion
    amount=nftoken_amount,  # 1 XRP = 1,000,000 drops
```